app/signin/page.tsx

```diff
@@ -0,0 +1,11 @@
+import Auth from "@/src/components/Auth/Auth";
+import AuthHookForm from "@/src/components/Auth/AuthHookForm";
+
+export default function signinPage() {
```

Comment:
next.js에서 export default를 지켜줘야하고, 컴포넌트명은 상관이 없기는 하지만 일반적인 컴포넌트명 컨벤션을 따라주면 좋겠네요!
```suggestion
export default function SignInPage() {
```